torch/onnx/__init__.py

```diff
@@ -288,8 +288,10 @@ def forward(self, x):
             If a custom opset is referenced by ``model`` but not mentioned in this dictionary,
             the opset version is set to 1.
 
-        enable_onnx_checker (bool, default True): If True the onnx model checker will be run
-            to ensure the exported model is a valid ONNX model.
+        enable_onnx_checker (bool, default True): [DEPRECATED. Backend logic is working as it's True.]
```

Comment:
This documentation is not accurate.
Should be something like:

`enable_onnx_checker (bool): Deprecated, ignored. Will be removed in next release.`